limbo/limbo.py

```diff
@@ -23,22 +23,28 @@
 
 logger = logging.getLogger(__name__)
 
+
 class InvalidPluginDir(Exception):
     def __init__(self, plugindir):
         message = "Unable to find plugin dir {0}".format(plugindir)
         super(InvalidPluginDir, self).__init__(message)
 
+
 def init_log(config):
     loglevel = config.get("loglevel", logging.INFO)
-    logformat = config.get("logformat", '%(asctime)s:%(levelname)s:%(name)s:%(message)s')
+    logformat = config.get("logformat",
+                           '%(asctime)s:%(levelname)s:%(name)s:%(message)s')
     if config.get("logfile"):
-        logging.basicConfig(filename=config.get("logfile"), format=logformat, level=loglevel)
+        logging.basicConfig(
+            filename=config.get("logfile"), format=logformat, level=loglevel)
```

Comment:
weird that it chose to put the first param on the same line as the paren in L35 but did the opposite for L39...